markdown/1.x/en/core-event-aggregator.md

```diff
@@ -1,12 +1,10 @@
-The `FFCommunicationEventAggregator` (**factfinder.communication.FFCommunicationEventAggregator**) is used to query 
-the FACT-Finder APIs with an event based approach. You can also hook into the communication pipeline.
+The **FFCommunicationEventAggregator** (`factfinder.communication.FFCommunicationEventAggregator`) is used to query the FACT-Finder APIs with an event based approach. You can also hook into the communication pipeline.
```

Comment:
```suggestion
The **FFCommunicationEventAggregator** (`factfinder.communication.FFCommunicationEventAggregator`) is used to query the FACT-Finder API's with an event based approach. You can also hook into the communication pipeline.
```

Reply:
Without `'` seems to be more common.
https://en.wiktionary.org/wiki/API
https://www.wordhippo.com/what-is/the-plural-of/API.html

Reply:
```suggestion
The **FFCommunicationEventAggregator** (`factfinder.communication.FFCommunicationEventAggregator`) is used to query the FACT-Finder API with an event based approach. You can also hook into the communication pipeline.
```

Reply:
I would use the _genitive '_. Whose query are you using?

If meant in plural I would use _APIs'_.

But if you are sure I trust your english ;)

Reply:
We can even switch to just `API`

Reply:
By the way, if we were using the genitive here, it would be `FACT-Finder's API`.

Reply:
yep, I like the last one most. But never mind for this one - I trust your english ;)